query/control/controller.go

```diff
@@ -166,6 +166,7 @@ func New(config Config) (*Controller, error) {
 	if err != nil {
 		return nil, errors.Wrap(err, "invalid controller config")
 	}
+	builtin.Initialize()
```

Comment:
Not needed, it now runs in the "command" code to avoid double-initialization in the flux test harness

Reply:
fixed! thanks. This was a semantic merge conflict.